services/bugmon/launch.sh

```diff
@@ -15,6 +15,10 @@ function tc-get-secret () {
   TASKCLUSTER_ROOT_URL="${TASKCLUSTER_PROXY_URL-$TASKCLUSTER_ROOT_URL}" retry taskcluster api secrets get "project/fuzzing/$1"
 }
 
+if [ -v "$FORCE_CONFIRM" ]; then
```

Comment:
You should use `[[ ]]`, especially if using a bash-ism like `-v`.

Also it's `-v varname` ..so you should either use `[[ -v FORCE_CONFIRM ]]` or `[[ -n "$FORCE_CONFIRM" ]]`